provider/aws.go

```diff
@@ -359,11 +365,16 @@ func (p *AWSProvider) newChange(action string, endpoint *endpoint.Endpoint) *rou
 	}
 
 	if isAWSLoadBalancer(endpoint) {
+		evalTargetHealth := p.evaluateTargetHealth
+		if _, ok := endpoint.ProviderSpecific[ProviderSpecificEvaluateTargetHealth]; ok {
+			evalTargetHealth = endpoint.ProviderSpecific[ProviderSpecificEvaluateTargetHealth] == "true"
+		}
```

Comment:
when writing records we use the value from the provider specific field or use the global default (that means: the default will only apply to newly created records).